Makefile

```
@@ -2,4 +2,8 @@
 
```

```suggestion
.PHONY: quality style
```
On line 1, I think maybe `style` should be added here.